WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -125,6 +125,10 @@ final class OrderListViewController: UIViewController, GhostableViewController {
     ///
     private var freeTrialBannerPresenter: FreeTrialBannerPresenter?
 
+    /// Notice presentation handler
+    ///
+    private var noticePresenter: NoticePresenter?
```

Comment:
If we are going to use the `DefaultNoticePresenter`, could we initialize it here as `private let noticePresenter: NoticePresenter = DefaultNoticePresenter()`

Reply:
Sounds good to me! So we can remove its optionality. Changed here: https://github.com/woocommerce/woocommerce-ios/pull/9730/commits/5224ad8a25c58058eb41c6ef4f8f0e8f0b9cf503